AK/Time.h

```diff
@@ -41,6 +42,12 @@ constexpr bool is_leap_year(int year)
     return year % 4 == 0 && (year % 100 != 0 || year % 400 == 0);
 }
 
+#ifndef KERNEL
+Optional<struct tm> convert_formatted_string_to_timespec(StringView input, StringView format, size_t&, size_t&);
```

Comment:
Since the `StringView`s are named already, the so-far unnamed `size_t&` should probably get a name as well.